src/main/java/uk/gov/ons/ctp/response/action/scheduled/plan/ActionPlanJobExecutor.java

```diff
@@ -69,27 +57,11 @@ private void createAndExecuteActionPlanJobs(final ActionPlan actionPlan) {
     }
 
```

Comment:
I prefer not to log anything if everything's working as it should be, because it's just log spam/bloat. In fact the Action service spams the logs and has managed to create 3.6 million useless rows in the ActionPlanJob table - we can get rid of that code and table, but it seemed a bit beyond the scope of the ticket.

Reply:
yeah i'd agree with nick on this one, at most it may be worth a trace/debug level logline to help devs follow the code through